doc/api/esm.md

```diff
@@ -687,12 +691,39 @@ provided via a `--experimental-loader ./loader-name.mjs` argument to Node.js.
 When hooks are used they apply to the entry point and all `import` calls. They
 won't apply to `require` calls; those still follow [CommonJS][] rules.
 
+Loaders follow the pattern of `--require`:
+
+```console
+node \
+  --experimental-loader unpkg \
+  --experimental-loader http-to-https \
+  --experimental-loader cache-buster
+```
+
+These are called in the following sequence: `cache-buster` calls
+`http-to-https` which calls `unpkg`. Or in terms of simple functions,
+`cacheBuster(httpToHttps(unpkg(input)))`.
```

Comment:
Perhaps this is not as clear as it could be, but I think it is accurate: in loaders-world, `https-to-https::resolve()` literally is called by `cache-buster::resolve()`.

The bit about results being passed I think is less clear. Perhaps I should remove the `cacheBuster(httpToHttps(unpkg(input)))` altogether since it's not 100% how it works?

I'm not married to this wording, but I think the suggestion is not clearer.

> […] loaders are invoked in the order they are presented on the command line.

This is not true (and it's the confusing part): they are invoked in **reverse** order of how they were passed in the command-line.

Reply:
> This is not true (and it's the confusing part): they are invoked in reverse order of how they were passed in the command-line.

That's not what this example shows: `unpkg` is the first command-line argument and `unpkg(input)` is the first function to be called too.

Reply:
I think removing `cacheBuster(httpToHttps(unpkg(input)))` is the best thing to do, because it's this syntax that makes it look reversed.